pkg/resources/view_grant.go

```diff
@@ -162,10 +162,10 @@ func ReadViewGrant(data *schema.ResourceData, meta interface{}) error {
 	var builder snowflake.GrantBuilder
 	if futureViews {
```

Comment:
i would refactore `futureViews` to `futureViewsEnabled` to make it obvious it is a boolean value. I would then, instead of the if/else would just `return readGenericGrant(data, meta, builder, futureViewsEnabled, ValidViewPrivileges)`